hazelcast/src/main/java/com/hazelcast/spi/OperationService.java

```diff
@@ -117,6 +117,15 @@
 
     <E> InternalCompletableFuture<E> invokeOnPartition(String serviceName, Operation op, int partitionId);
 
+    /**
+     * Executes an operation on a partition.
+     *
```

Comment:
Just a pickle: since there's no partitionId in the arguments of this method it may be worth mentioning how it know on which partition to invoke the operation on, like 
```Executes an operation on a partition. 
Partition that the operation is invoked on comes from Operation.getPartitoinId()`
```